addons/waypoint/CfgWaypoints.cfg

```diff
@@ -0,0 +1,11 @@
+class CfgWaypoints {
+    class AF_Waypoints {
+        displayName = "ArmaForces";
+        class Move_vehicle {
+            displayName = CSTRING(Vehicle_Move_Name);
```

Comment:
`Vehicle Move` or `Move Vehicle`, stringtable naming is inconsistent with class. Also use camel case for this (VehicleMove or MoveVehicle)

Reply:
Actually that's `PascalCase`, not `camelCase`.